docs/docs/webui.md

```diff
@@ -19,7 +20,7 @@ There are also two included example YAML files that make use of this configurati
 * [config/local_scheduler.yml](/mesos/chronos/blob/master/config/local_scheduler.yml#L4)
 * [config/local_scheduler_nozk.yml](/mesos/chronos/blob/master/config/local_scheduler_nozk.yml#L7)
 
-## Build Requirements
+### Build Requirements
 
```

Comment:
It seems that the issue is closed (#42) some time ago so I wouldn't mention it.  @mlunoe, did you encounter any out of memory issue when you built the Chronos UI? Just want to double check.